### PR TITLE
Open Source Places Blog Post

### PR DESCRIPTION
**Pull request by eneemann**
PR for blog post "Introducing OpenStreetMap Places"

### CODE REVIEW

**Comment by eneemann** [comment]
The contact info isn't populating yet, but should once PR #1969 is merged.  At that time, I'll also add links in the blog post to the data page.

**Comment by steveoh** [comment]
If you added the contact info in an atomic commit, you can cherry pick it

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+All kinds!  There are businesses, restaurants, churches, schools, banks, hotels, viewpoints, attractions, and more.  Take a look at this chart showing the counts of the 25 most popular types of locations.
+
+![OpenStreetMap Feature Counts]({% link images/osm_counts.png %} "OpenStreetMap Feature Counts"){: .flex .flex--center}
+
+### How can it be used?
+{: .text-left}
+
+We think there will be numerous potential use cases for such a large and varied dataset, for state and local agencies, citizens, and even researchers.  The data could be used to perform different analyses on certain business types or even specific businesses.  How many Beans and Brews locations are within a half-mile of Starbucks?  The businesses and landmarks could be used as a reference for 911 call-takers, first responders, or emergency managers.  This can be particularly helpful because we don't always know the address of our emergencies, but we usually know a point of interest or place name.  The data could even use it to quality control other datasets and look for features that we may be missing.  Are we missing any schools, fire stations, liquor stores, or cemeteries from our authoritative datasets?
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:32]: **issue**: this reads funny to me. 

> The data could even use it to quality control other datasets and look for features that we may be missing

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+### How can I contribute?
+{: .text-left}
+
+It's pretty easy, really.  All you need to do is create a free [OSM account](https://www.openstreetmap.org/user/new), login, start editing or creating places (iD is the default editor), and save your updates.  Then, the next time we run our python script to update this layer, your changes will be included and published in the SGID.  In addition to the default, web-based editor on openstreetmap.org ([iD](https://www.openstreetmap.org/edit?editor=id)), there are several desktop and mobile (Android and iOS) editors can be used, including, [JOSM](https://josm.openstreetmap.de/), [Vespucci](https://vespucci.io/), [GoMap!!](https://apps.apple.com/app/id592990211), and the [ArcGIS OpenStreetMap Editor](https://github.com/Esri/arcgis-osm-editor) (ArcGIS Desktop 10+ only).  A larger list of editors is maintained on the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Comparison_of_editors).  If creating an OSM account and using an editor are too cumbersome, you can also submit businesses anonymously with a simple form entry and pin-location at [OnOSM](http://onosm.org).  Your submissions through OnOSM will be reviewed and added by an OSM user.
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:37]: **issue**: this reads funny also

> there are several desktop and mobile (Android and iOS) editors can be used,

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+![OSM Handshake]({% link images/osm_handshake.png %} "OSM Handshake"){: .flex .flex--center}
+
+### How does it differ from other SGID data?
+{: .text-left}
+
+- It's **not** authoritative - due to the nature of crowdsourced data, we can't guarantee that it's perfect or accurate
+- It has a different license - instead of our usual [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
+- It's more dynamic - it gets updated frequently, is constantly growing, and anyone can contribute to it
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:46]: **thought**: this makes it sounds like SGID data isn't updated frequently and isn't constantly growing

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+### How is it similar to other SGID data?
+{: .text-left}
+
+- It may include locations that are represented in other, authoritative, SGID datasets
+   - Examples: schools, parks, liquor stores, libraries, fire stations, etc.
+- It's available in a variety of ways and data formats
+   - ArcGIS Online services
+   - OpenData
+   - OpenSGID - our Postgres database in the Google Cloud
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:58]: **nitpick**: remove implementation details
```suggestion
   - OpenSGID
```

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+   - Downloads in your favorite data format (shapefile, file geodatabase, GeoJSON, KML, CSV)
+
+
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:61]: **nitpick**:  remove empty newline
```suggestion
```

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+   - centre (instead of center, ex: sports_centre, arts_centre, garden_centre)
+   - theatre (instead of theater)
+   - caravan_site (instead of rv_park or trailer_park
+   - archaeological (instead of archeological)
+   - chemist (instead of pharmacy)
+   - tyres (instead of tires)
+- There are two types of addresses provided in the data:
+   1. The address contained in the OSM data (`osm_addr`).  These were input by OSM contributors, but only a small percentage of the points come with an address (~24%).
+   1. The address of the nearest UGRC address point (`ugrc_addr`).  These are not official addresses of the businesses or points of interest, but provide a very close address for a greater percentage of points (~59%).  They are the nearest address (within 25m) in the [UGRC Address Points](https://opendata.gis.utah.gov/datasets/utah-address-points/explore) dataset.  The `addr_dist` field provides the distance (in meters) from the OSM point to the nearby address point.
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:75]: **thought**: should we link internally to the address points?

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+### What are the details on how this data is created?
+{: .text-left}
+
+The OSM Places layer is created by a Python script that pulls statewide OSM data from a nightly archive provided by [Geofabrik](https://www.geofabrik.de/data/download.html).  The archive data contains nearly 20 shapefiles, some that are relevant and some that aren't.  So, the OSM Places layer is built by filtering the data in those shapefiles down to a single point dataset with specific categories and attributes.  Additional de-duplication and spatial filtering are done along the way, to ensure the data is as clean as possible.  Then, additional fields are created and assigned from UGRC's SGID data (county, city, zip, nearby address, etc.) via point-in-polygon analysis.  Finally, additional attributes (OSM address, open hours, cuisine, etc.) are pulled from the [Overpass API](https://wiki.openstreetmap.org/wiki/Overpass_API) and joined to the filtered data using the 'osm_id' field as the join key.  Additional information can be found on the OpenStreetMap Places data page.
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:89]: **thought**: should this be a link?

> Additional information can be found on the OpenStreetMap Places data page

File: _posts/2022-03-10-introducing-openstreetmap-places.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+If you really want to get into the _nitty gritty_ details, come to the UGRC talk the [2022 UGIC Conference](https://ugic.org/uncategorized/ugic-registration-open/) or keep your eyes peeled :eyes: for a future blog post.
```
Comment (steveoh) [inline _posts/2022-03-10-introducing-openstreetmap-places.md:91]:
```suggestion
If you really want to get into the _nitty gritty_ details, come to my [2022 UGIC Conference](https://ugic.org/uncategorized/ugic-registration-open/) presentation or keep your eyes peeled 👀 for a future blog post.
```

**Review by steveoh — CHANGES_REQUESTED** [review]
you're still an old fashion double spacer? It's even an error in [microsoft word](https://twitter.com/APStylebook/status/1254786139539427329?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1254786139539427329%7Ctwgr%5E%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fwww.patrickkphillips.com%2Fgrammar%2Ftwo-spaces-after-a-period-or-just-one-please-get-it-right%2F) and the AP agrees!

@gregbunce should see that tweet also 😂🤣😇

File: _posts/2022-03-10-introducing-openstreetmap-places.md (hunk 2 above)
Comment (gregbunce) [reply _posts/2022-03-10-introducing-openstreetmap-places.md:37]: don't succumb to steve's single-spaced bullying. haha. just kidding. I'm actually trying to reprogram my thumb to tap the spacebar once between sentences. it's not easy.

File: _posts/2022-03-10-introducing-openstreetmap-places.md (hunk 3 above)
Comment (gregbunce) [reply _posts/2022-03-10-introducing-openstreetmap-places.md:46]: agreed, that's the only thing that stood out to me on this nicely-done post. I know what you're saying in that third bullet, just thinking it could be worded differently.

File: _posts/2022-03-10-introducing-openstreetmap-places.md (hunk 7 above)
Comment (eneemann) [reply _posts/2022-03-10-introducing-openstreetmap-places.md]: Yes, but that link was breaking the netlify build because the page doesn't exist yet.  I'll add it after the data page gets merged.

File: _posts/2022-03-10-introducing-openstreetmap-places.md (hunk 6 above)
Comment (eneemann) [reply _posts/2022-03-10-introducing-openstreetmap-places.md]: Yeah, I was thinking about that too.

**Comment by eneemann** [comment]
> If you added the contact info in an atomic commit, you can cherry pick it

Yeah, it's this commit: dfcb1402b91786f321043d1d7a4c3e2808b8e0eb on the Data Page PR #1969, which could probably be merged whenever we're ready.  Just gotta update the AGOL ID when it's available.

**Comment by steveoh** [comment]
> Just gotta update the AGOL ID when it's available. 

Should be in there

**Comment by stdavis** [comment]
@eneemann I rebased this branch against master so that it could find the newly created data page.

**Comment by steveoh** [comment]
Don't forget to update the dates

File: _posts/2022-03-10-introducing-openstreetmap-places.md (hunk 3 above)
Comment (eneemann) [reply _posts/2022-03-10-introducing-openstreetmap-places.md]: This was changed to "Any user can contribute to it - the data can be quickly improved and grown by OSM users"

File: _posts/2022-03-10-utah-sgid-statewide-roads-data-layer-updates-03102022.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (steveoh) [inline _posts/2022-03-10-utah-sgid-statewide-roads-data-layer-updates-03102022.md:1]: **issue**: this file should not be in here...

File: _posts/2022-03-21-introducing-open-source-places.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+  - openstreetmap
+  - OSM
+  - places
+  - points of interest
+  - businesses
+---
+
+![OpenSourcePlaces]({% link images/map-poi-256.png %} "https://viglino.github.io/font-gis/?q=poi&fg=map-poi"){: style="float: right"}
+
+There's an exciting new data layer that has been recently added to the State Geographic Information Datasource (SGID): [Open Source Places]({% link data/society/open-source-places/index.html %}). This is a brand-new offering that is created from open source data and is intended to represent places of interest in the state of Utah. These may include businesses, restaurants, places of worship, airports, parks, schools, event centers, apartment complexes, hotels, car dealerships, etc., etc., …almost anything that you can find in OpenStreetMap (OSM). There are over 23,000 features in the data, and the best part of all? You can make updates and contribute directly to this layer through [OpenStreetMap](http://openstreetmap.org)! (more on that later)
```
Comment (steveoh) [inline _posts/2022-03-21-introducing-open-source-places.md:20]: **suggestion**: add a link to porter?

```suggestion
There's an exciting new data layer that has been recently [added](https://github.com/agrc/porter/issues/180) to the State Geographic Information Datasource (SGID): [Open Source Places]({% link data/society/open-source-places/index.html %}). This is a brand-new offering that is created from open source data and is intended to represent places of interest in the state of Utah. These may include businesses, restaurants, places of worship, airports, parks, schools, event centers, apartment complexes, hotels, car dealerships, etc., etc., …almost anything that you can find in OpenStreetMap (OSM). There are over 23,000 features in the data, and the best part of all? You can make updates and contribute directly to this layer through [OpenStreetMap](http://openstreetmap.org)! (more on that later)
```

File: _posts/2022-03-21-introducing-open-source-places.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+You can get involved in the OSM community by checking out upcoming events on the [OSM Calendar](https://osmcal.org/?in=United%20States).  The calendar will list events across the US, including those right here in Utah.  The [OSM Slack community](https://slack.openstreetmap.us/) is also open to anyone, with a specific `#local-utah` channel for users and events in the state.
+
+![Epic Handshake]({% link images/os_handshake.png %} "Epic Handshake"){: .flex .flex--center}
+
+### How does it differ from other SGID data?
+{: .text-left}
+
+- It's **not** authoritative - due to the nature of crowdsourced data, we can't guarantee that it's perfect or accurate
+- It has a different license - instead of our usual [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
```
Comment (steveoh) [inline _posts/2022-03-21-introducing-open-source-places.md:49]: **suggestion**: link to our license page 

```suggestion
- It has a different license - instead of our usual [CC BY 4.0]({% link about/policy/license-disclaimer/index.md %}}) license, it carries the [ODbL 1.0](https://opendatacommons.org/licenses/odbl/summary/) license
```

File: data/911/index.html [hunk 12]
```diff
@@ -17,7 +17,7 @@
     - url: https://drive.google.com/a/utah.gov/uc?id=1Sp6v4FmNV5fhdfEWPTxPGHMbnz-yeY47&export=download
       label: 'UtahNG911GIS: File Geodatabase'
   updates:
```
Comment (steveoh) [inline data/911/index.html:19]: **issue**: this shouldn't be included in this pr

File: data/cadastre/parcels/index.html [hunk 13]
```diff
@@ -163,7 +163,7 @@ <h5 id="UtahParcels-downloads">Downloads</h5>
         </tr>
         <tr>
           <td>Kane</td>
-          <td><a href="https://opendata.gis.utah.gov/datasets/utah-kane-county-parcels">January 2022</a></td>
```
Comment (steveoh) [inline data/cadastre/parcels/index.html:166]: **issue**: this shouldn't be included. What did you do to pick up these commits?

File: data/society/openstreetmap-places/index.html [hunk 14]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (steveoh) [inline data/society/openstreetmap-places/index.html:1]: **issue**: this should be deleted right?

**Review by steveoh — CHANGES_REQUESTED** [review]
This is looking good. But there are a few problems. 

- You've got the cadastre and transportation commits that need to be removed
- You've got old images that aren't being used

File: data/society/openstreetmap-places/index.html (hunk 14 above)
Comment (eneemann) [reply data/society/openstreetmap-places/index.html:1]: good catch

**Comment by eneemann** [comment]
Replace by PR #1979
